### PR TITLE
Add initial setup

### PR DESCRIPTION
**Pull request by ajfloeder**
Adding an initial setup markdown file with some information for immediate use installing the early access system.

### CODE REVIEW

**Review by roehrich-hpe — APPROVED** [review]

File: initial-setup.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+Taking these in reverse order using option 3 above which allows UDEV settings within the host OS to remain unchanged from the default, one would need to start the `dmsetup` command on a separate thread because the LVM create/remove command waits for the UDEV cookie. This opens too many error paths, so it was rejected.
+
+Option 2 allows UDEV settings within the host OS to remain unchanged from the default, but the use of UDEV within production Rabbit systems is viewed as unnecessary because the host OS is PXE-booted onto the node vs loaded from an device that is discovered by UDEV.
+
+Option 1 above is what we chose to implement because it is the simplest. The following sections discuss this setting.
+
+In order for LVM commands to run within the container environment on a Rabbit, the following change is required to the `/etc/lvm/lvm.conf` file on Rabbit.
+
+```bash
+grep udev_[r:s] /etc/lvm/lvm.conf
```
Comment (NateRoiger) [inline initial-setup.md:24]: Why the 'r'? Or the regex at all. Why not just give them a one-liner to patch the lvm.conf file

File: initial-setup.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Initial Setup Instructions
+
+Instructions for the initial setup of a Rabbit are included in this document.
+
+## LVM Configuration on Rabbit
+
+Running LVM commands (lvcreate/lvremove) on a Rabbit to create logical volumes is problematic if those commands run within a container. Rabbit Storage Orchestration code contained in the `nnf-node-manager` Kubernetes pod executes LVM commands from within the container. The problem is that the LVM create/remove commands wait for a UDEV confirmation cookie that is set when UDEV rules run within the host OS. These cookies are not sync'ed with the containers where the LVM commands execute.
```
Comment (NateRoiger) [inline initial-setup.md:7]: I'm not sure the backstory is needed here - could you maybe make it foldable?

**Comment by NateRoiger** [comment]
We also need a valid SystemConfiguration. We must ensure the NNF-NODE-NAME is present in the config as well I think.

**Comment by NateRoiger** [comment]
Nice work! This will be immensely helpful.

**Review by NateRoiger — APPROVED** [review]

**Review by billhpe — APPROVED** [review]

**Pull request merged**